src/Pos/CLI.hs

```diff
@@ -287,10 +287,9 @@ externalNetworkAddressOption na =
          <> Opt.showDefault
          <> maybe mempty Opt.value na
   where
-    helpMsg = "Ip and port of external address. "
-        <> "Please mind that you need to specify actual accessible "
-        <> "ip of host, at which node is run,"
-        <> " otherwise work of CSL is not guaranteed. "
+    helpMsg = "IP and port of external address. "
+        <> "Please make sure these IP and port (at which node is running) are actually accessible "
```

Comment:
s/at/on/
s/actually//g (actually has a different meaning, i can't find something to exchange it with tho)